pkg/sbproxy/notifications/handlers/broker_handler.go

```diff
@@ -161,7 +175,7 @@ func (bnh *BrokerResourceNotificationsHandler) OnUpdate(ctx context.Context, pay
 	log.C(ctx).Infof("Successfully found platform broker with name %s and URL %s.", existingBroker.Name, existingBroker.BrokerURL)
 
 	if existingBroker.BrokerURL != brokerProxyPath {
-		log.C(ctx).Errorf("Platform broker with name %s has an URL %s and is not proxified by SM. No update will be attempted", existingBroker.Name, existingBroker.BrokerURL)
+		log.C(ctx).Errorf("Platform broker with name %s has an URL %s and is not taken over by SM. No update will be attempted", existingBroker.Name, existingBroker.BrokerURL)
```

Comment:
well, if an update notification for broker that is blacklisted comes, what will happen ? Same for delete. Same for visibilities

Reply:
Update and Delete broker notifications will fall back to the case where we check the platform for the broker, but it won't be found, so nothing will happen:

https://github.com/Peripli/service-broker-proxy/blob/aac6c78095f2d61a4936cfbebf11964668b1b732/pkg/sbproxy/notifications/handlers/broker_handler.go#L172

https://github.com/Peripli/service-broker-proxy/blob/aac6c78095f2d61a4936cfbebf11964668b1b732/pkg/sbproxy/notifications/handlers/broker_handler.go#L236

Reply:
Addressed.